catalogue/webapp/components/WorkCard/WorkCard.js

```diff
@@ -0,0 +1,224 @@
+// @flow
+import NextLink from 'next/link';
+import styled from 'styled-components';
+import { type Work } from '@weco/common/model/work';
+import { classNames, spacing, font } from '@weco/common/utils/classnames';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import LinkLabels from '@weco/common/views/components/LinkLabels/LinkLabels';
+import TogglesContext from '@weco/common/views/components/TogglesContext/TogglesContext';
+import {
+  getPhysicalLocations,
+  getDigitalLocations,
+  getProductionDates,
+  getWorkTypeIcon,
+} from '@weco/common/utils/works';
+import { trackEvent } from '@weco/common/utils/ga';
+import { workUrl } from '@weco/common/services/catalogue/urls';
+
+type Props = {|
+  work: Work,
+  query: ?string,
+  page: ?number,
+  workType: string[],
+  itemsLocationsLocationType: string[],
+|};
+
+const Container = styled.div`
+  ${props => props.theme.media.medium`
+    display: flex;
+  `}
+`;
+const Details = styled.div`
+  ${props => props.theme.media.medium`
+    flex-grow: 1;
+  `}
+`;
+const Preview = styled.div`
+  margin-top: ${props => props.theme.spacingUnit * 2}px;
+  ${props => props.theme.media.medium`
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: '178px';
+    height: '178px';
+    margin-top: 0;
+  `}
+`;
+
+const WorkCard = ({
+  work,
+  query,
+  page,
+  workType,
+  itemsLocationsLocationType,
+}: Props) => {
+  const digitalLocations = getDigitalLocations(work);
+  const physicalLocations = getPhysicalLocations(work);
+  const productionDates = getProductionDates(work);
+  const workTypeIcon = getWorkTypeIcon(work);
+  return (
+    <div
+      className={classNames({
+        'border-color-pumice': true,
+        'border-top-width-1': true,
+      })}
+    >
+      <NextLink
+        href={
```

Comment:
Can be updated to `{...workUrk({
            id: work.id,
            query,
            page,
            workType,
            itemsLocationsLocationType,
          })}` probably?

Reply:
workUrk?
Do you mean workIrk?

Reply:
`workLurk`